src/theme.js

```diff
@@ -9,7 +9,7 @@ const theme = {
     yellow: ['#ffefc5', '#ffd972', '#fcc440', '#ee892b']
   },
   fonts: {
-    sans: `"Inter UI", -apple-system, BlinkMacSystemFont, 'San Francisco', 'Helvetica Neue', Arial, sans-serif`,
+    sans: `"Inter", "Inter UI", -apple-system, BlinkMacSystemFont, 'San Francisco', 'Helvetica Neue', Arial, sans-serif`,
```

Comment:
Just a heads up @g-a-v-i-n, this is just for development purposes. Basically, I have it here so it's more convenient to make/test changes in the library (instead of having to update the tokens repo + publish new version + install new version).

Need to update the fonts [here](https://github.com/urbit/indigo-tokens/blob/master/index.js#L12).

I do export the theme... so I realize that's a little confusing. Maybe should name it something like `DEVELOPMENT_THEME` a la React's `UNSAFE_` methods to discourage usage in a production app.